### PR TITLE
Fixed "Copy current receive event"

### PR DESCRIPTION
**Pull request by oleg68**
Resolves: #985

This PR changes the behaviour of the "Copy current receive event" button in the Send page of the Midi Event Editor.

The changes:
- Now the event is copied even the input device is either not selected or does not have an output device specified. But in these cases the Device fiels is set to ``Any Device``
- Does not crash if the source input event has ``Any Channel``. In this case the channel becomes `1`
- OK is possible only all sending events have correct `Device` values. ``Any device`` is not valid and the user should select a device manually
- Added information about this button to the help

Also there are small changes in the layout of the Midi Event Editor.

This PR is based on #997, so most of changed files will go after merging #997

### CODE REVIEW

**Comment by oleg68** [comment]
@larspalo @rousseldenis could you look at this PR?

File: src/grandorgue/midi/MIDIEventDialog.cpp [hunk 1]
```diff
@@ -73,14 +74,44 @@ void MIDIEventDialog::RegisterMIDIListener(GOMidi *midi) {
...
+  wxWindow *notValidTab = NULL;
+  wxString errMsg;
+
+  if (!notValidTab && m_sendPage && !m_sendPage->Validate(errMsg))
+    notValidTab = m_sendPage;
+  if (
+    !notValidTab && m_sendDivisionPage && !m_sendDivisionPage->Validate(errMsg))
+    notValidTab = m_sendPage;
+
+  // Distplay the tab that has problems
```
Comment (rousseldenis) [inline src/grandorgue/midi/MIDIEventDialog.cpp:87]: Display

**Review by rousseldenis — APPROVED** [review]
Code review. Nitpicking

File: src/grandorgue/midi/MIDIEventDialog.cpp (hunk 1 above)
Comment (oleg68) [reply src/grandorgue/midi/MIDIEventDialog.cpp]: Fixed

**Comment by oleg68** [comment]
@larspalo could you approve this?

**Review by larspalo — APPROVED** [review]

**Pull request merged**